collections/Don/MS_Don_e_250.xml

```diff
@@ -71,10 +76,441 @@
+                     </handDesc>
+                     <decoDesc>
+                        <decoNote type="rubrication">Headings, paragraph marks, and marginal comments in red. Certain words and phrases underlined in red. Majuscules stroked in red. </decoNote>
+                     </decoDesc>
+                  </physDesc>
+                  <history>
+                     <origin>
+                        <origDate calendar="Gregorian" notAfter="1500" notBefore="1440">15th century, middle or second half</origDate>
+                        <origPlace>
+                           <country key="place_7000084">German</country>, <region>west</region>
+                        </origPlace>
+                     </origin>
+                  </history>
+               </msPart>
+               <msPart xml:id="MS_Don_e_250-partIV">
+                  <msIdentifier>
+                     <altIdentifier type="partial">
+                        <idno type="part">MS. Don. e. 250 - lower endleaf, fol. 160</idno>
+                     </altIdentifier>
+                  </msIdentifier>
+                  <msContents>
+                     <textLang mainLang="la" otherLangs="de">Latin and German</textLang>
+                     <msItem n="14">
+                        <note>(fol. 160r) blank.</note>
+                        <note>Unidentified Latin-German text by a single hand on fol. 160v: ‘[…] dulcedo spiritus sine est clauis  [Considera] das mynneclich inkeren [zu above in as alternative reading] in den willen sins himmelschen vatters / Die hiczigen trehe […] pro me  / das anedehtige Innig gebett das als starck  was das sich alle sine kreffe von vßen vnd ynne  sich zu grunde liessende dum dixit quoniam fiat volutas tua  das sich selber geben gancz dich mir In cora re (?) dixit  / Item die fruht suis lydens in cruce / lauabor […]’.</note>
+                     </msItem>
+                  </msContents>
+                  <physDesc>
+                     <objectDesc form="codex">
+                        <supportDesc material="chart">
+                           <support>Paper, folded in octavo. Watermark, part of an anchor, Piccard II or III 1–132.</support>
+                        </supportDesc>
+                     </objectDesc>
+                  </physDesc>
+                  <history>
+                     <origin>
+                        <origDate calendar="Gregorian" notAfter="1500" notBefore="1440">15th century, middle or second half</origDate>
+                        <origPlace>
+                           <country key="place_7000084">German</country>, <region>west</region> (?)
+                        </origPlace>
+                     </origin>
+                  </history>
+               </msPart>
             </msDesc>
          </sourceDesc>
       </fileDesc>
       <revisionDesc>
+         <change when="2021-09">New description encoded from description by Jonas Hermann.</change>
```

Comment:
Your name is missing.

Reply:
agree that the encoder's name should be given here if it is not already given in `source` - but isn't it redundant if already in `source`?